docs/articles/streams/integration.md

```diff
@@ -735,3 +518,306 @@ var flow = Flow.FromProcessor(()=> createProcessor(materializer));
 ```
 
 Please note that a factory is necessary to achieve reusability of the resulting `Flow`.
+
+####Implementing Reactive Streams Publisher or Subscriber
+
+As described above any Akka Streams ``Source`` can be exposed as a Reactive Streams ``Publisher`` 
+and any ``Sink`` can be exposed as a Reactive Streams ``Subscriber``. Therefore we recommend that you 
+implement Reactive Streams integrations with built-in stages or [custom stages](customstreamprocessing.md).
+
+For historical reasons the `ActorPublisher` class and `ActorSubscriber` class traits are
```

Comment:
remove `traits` , they are kind of interfaces in .net but aren't relevant here since one would extend the abstract classes instead of the interface.